### DISCUSSION TITLE
Card

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#240] [EXPERIMENTAL: Components, pages and patterns]
# Existing

- [Summary card](https://github.com/alphagov/govuk-design-system-backlog/issues/210) - On the GOV.UK Design System backlog
- [GOV.UK summary card examples](https://govuk-summary-card.glitch.me/) - Put together by DfE in order to make this component available so people can try it out and see how it’s put together.
- [Card component working Google doc](https://docs.google.com/document/d/17595llJ9O0FcIm0XyH_4cLYEz45HsjNFfcQgdTTv-xs/edit?usp=sharing)

![Homepage (Non-managers)](https://user-images.githubusercontent.com/6122118/127007751-efd9d308-bfa2-4c12-9bde-97b7cf025352.png)

**Comment by Ed209uk** [comment]
This component replaced our initial 'card' designs which only had icons on there. They were not user-friendly or scaleable so had to go. The description on each card has really helped users to know what each area will lead to and tested well. I'm going to get hold of the feedback from user testing we did and add it to here.

We've also used this component for sub-pages, under the main home page (without the search for prisoner) and that has worked really well. This component also works well at various different view port sizes (including mobile).

**Reply by JSJohal** [reply]
Just to add to Eddie's post, here are some screenshots from across DPS of the cards component being used.

**DPS homepage - navigation landing page**

![image](https://user-images.githubusercontent.com/22663194/131975915-ec01fc15-2fb9-4174-b30e-1ad4d56ed669.png)

**DPS - a layer down in navigation**

![image](https://user-images.githubusercontent.com/22663194/131976014-b40b611a-d1ff-49f4-a907-dabe704a1587.png)

**Comment by olivia-todd** [comment]
We have used the card component on the Content Hub service and have also created 3 new variants due to our user needs.


### **Basic card**

As seen in the NHS design system: https://service-manual.nhs.uk/design-system/components/card. A basic card is a card without a link or an image. We created this component as a way to let people scan and read relevant information that can't be interacted with (yet).

![Basic card](https://user-images.githubusercontent.com/55841349/131124054-11dc18c1-37d6-488a-930a-47f4fc274bbf.jpg)


### **Card with an image**

As seen in the NHS design system: https://service-manual.nhs.uk/design-system/components/card. We created a card with an image as our research has shown that people in prison find it easier to navigate our service when they have additional visual cues such as images or icons.

![Basic card and card with image](https://user-images.githubusercontent.com/55841349/131124040-5c4c9532-5f1e-475b-86c2-97e1c6738b0a.jpg)


### **Expandable card - EXPERIMENTAL**

This is an experimental design where we have combined a **clickable card** with an **accordion** component, and which we're calling an **expandable card** for the time being.

We created this variant because our research showed that some prisoners had concerns over certain personal data (e.g. finances, names of family members) being visible by default whilst in a prison or shared cell environment. We are experimenting with an expandable version of a card (using a pattern from the [accordion](https://design-system.service.gov.uk/components/accordion/) component) which allows users to show and hide sections of content on a page to give them more control over when they want this data to be visible on the page.

![Expandable card - normal](https://user-images.githubusercontent.com/55841349/131123915-44b05a75-718f-4e73-82aa-7e6b3a6d2a0d.jpg)

If there are multiple expandable cards in the same section, we are experimenting with an “open all” link (taken again from the accordion component) so that users can open or close all expandable cards more quickly with one click rather than several clicks, but still retain the ability to open just one card at a time if they wish.

![Expandable card - expanded](https://user-images.githubusercontent.com/55841349/131123924-ff426ba6-e6ff-404e-b7e8-ebf4a349ca71.jpg)

The expandable card tested well in usability testing and so far has been received well in the 2 prisons where we are live. We are monitoring feedback on this experimental component and will update this discussion with any further feedback we receive.

**Comment by JSJohal** [comment]
Coded examples of the cards as used in DPS in the prototype kit:

HTML: https://github.com/ministryofjustice/hmpps-rap-prototype/blob/master/app/views/v6/dps-home.html

CSS: https://github.com/ministryofjustice/hmpps-rap-prototype/blob/master/app/assets/sass/components/_cards.scss

**Comment by helennickols** [comment]
The card component has been published on the MOJ Design System. This is to make it easier to find and use.  

https://design-patterns.service.justice.gov.uk/components/card/

**Comment by helennickols** [comment]
The 'card' colour is not compatible with GOV.UK Frontend v6.0.0. Because this is an experimental component this change will be made when it's brought to the 'official' status.
